includes/html/api_functions.inc.php

```diff
@@ -988,9 +988,8 @@ function list_alerts(\Illuminate\Http\Request $request)
     }
 
     $order = 'timestamp desc';
-
     
-    $alert_rule = $_GET['alert_rule'];
+    $alert_rule= $request->get('alert_rule');
```

Comment:
just a little picky, but please change `$alert_rule= $request->get('alert_rule');` to `$alert_rule = $request->get('alert_rule');`